### DISCUSSION TITLE
shadow variable

### DISCUSSION THREAD

**Discussion by maddin200** [bulletphysics/bullet3#3654] [General]
bullet3-master/src/LinearMath/btQuickprof.cpp) (line 665):  same variable i (as well line 657).

**Comment by wduminy** [comment]
It is likely that there are many places in the code that uses `i` in the way that [line 665](https://github.com/bulletphysics/bullet3/blob/master/src/LinearMath/btQuickprof.cpp#L665) does.
 
How would one motivate fixing all of them? Is it worth the effort?
